lib/ebay_client/version.rb

```diff
@@ -1,3 +1,3 @@
 module EbayClient
-  VERSION = '0.2.0.981'
+  VERSION = '0.2.0.982'
```

Comment:
This is the version used for the gemspec, So I suggest you put the ebay version somewhere else so the ruby gem can be properly versioned with `bundle update`